interpreter/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala

```diff
@@ -650,8 +652,8 @@ class SpelExpressionSpec extends AnyFunSuite with Matchers with ValidatedValuesD
   }
 
   test("be able to type string concatenation") {
-    parse[Any]("12 + ''", ctx).toOption.get.returnType shouldBe Typed[String]
-    parse[Any]("'' + 12", ctx).toOption.get.returnType shouldBe Typed[String]
+    parse[Any]("12 + ''", ctx).toOption.get.returnType shouldBe Typed.fromInstance("12")
```

Comment:
This tests double logic of `should calculate values of operators on strings`. Shouldn't we remove them?